pkg/operation/botanist/controlplane.go

```diff
@@ -1054,64 +1058,101 @@ func (b *Botanist) DeployETCD(ctx context.Context) error {
 		hvpaEnabled = gardenletfeatures.FeatureGate.Enabled(features.HVPAForShootedSeed)
 	}
 
-	etcdConfig := map[string]interface{}{
+	defaultValues := map[string]interface{}{
 		"podAnnotations": map[string]interface{}{
-			"checksum/secret-etcd-ca":         b.CheckSums[v1beta1constants.SecretNameCAETCD],
-			"checksum/secret-etcd-server-tls": b.CheckSums["etcd-server-tls"],
-			"checksum/secret-etcd-client-tls": b.CheckSums["etcd-client-tls"],
+			"checksum/secret-etcd-ca":          b.CheckSums[v1beta1constants.SecretNameCAETCD],
+			"checksum/secret-etcd-server-cert": b.CheckSums[common.EtcdServerTLS],
+			"checksum/secret-etcd-client-tls":  b.CheckSums[common.EtcdClientTLS],
 		},
 		"hvpa": map[string]interface{}{
 			"enabled": hvpaEnabled,
 		},
 		"storageCapacity": b.Seed.GetValidVolumeSize("10Gi"),
 	}
 
-	etcd, err := b.InjectSeedShootImages(etcdConfig, common.ETCDImageName)
+	values, err := b.InjectSeedShootImages(defaultValues,
+		common.ETCDImageName,
+		common.ETCDBackupRestoreImageName,
+	)
 	if err != nil {
 		return err
 	}
 
+	values["prefix"] = common.GenerateBackupEntryName(b.Shoot.Info.Status.TechnicalID, b.Shoot.Info.Status.UID)
+
 	for _, role := range []string{common.EtcdRoleMain, common.EtcdRoleEvents} {
-		etcd["role"] = role
+		values["role"] = role
+		values["backupEnabled"] = false
```

Comment:
etcd charts no more have the. "backupEnabled" field. Please remove it.

Reply:
I use it in the chart in gardener to add the fields related to backup and store.

Reply:
Ahh sorry. Its there in etcd CRD but not in values.yaml. Will you please add it in values.yaml of etcd chart.

Reply:
Done